stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/search/SearchResultListView.kt

```diff
@@ -0,0 +1,72 @@
+package io.getstream.chat.android.ui.search
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.LinearLayoutManager
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.databinding.StreamSearchResultListViewBinding
+
+public class SearchResultListView : LinearLayout {
+
+    private val binding = StreamSearchResultListViewBinding.inflate(LayoutInflater.from(context), this)
+
+    public constructor(context: Context) : super(context) {
+        init(null)
+    }
+
+    public constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+        init(attrs)
+    }
+
+    public constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
+        context,
+        attrs,
+        defStyleAttr
+    ) {
+        init(attrs)
+    }
+
+    private val adapter = SearchResultListAdapter()
+
+    private fun init(attrs: AttributeSet?) {
+        parseAttrs(attrs)
+
+        orientation = VERTICAL
```

Comment:
NIT: This can be probably removed as it is already set in XML

Reply:
This is for the root LinearLayout, this still has to be set from code.

`SearchResultListView` is a vertical `LinearLayout`, which has a `TextView`("X results") and a `RecyclerView` inside it.